tests/installation/zdup.pm

```diff
@@ -89,7 +89,11 @@ sub run() {
                        done
                        [ -z \$dev ] || echo \"found dev \$dev with label \$label\"";
             # if that fails, e.g. if volume descriptor too long, just try /dev/sr0
-            $defaultrepo = "dvd:/?devices=\${dev:-/dev/sr0}";
+            # get all attached ISOs including addons' as zdup dup repos
+            my $srx = script_output("ls -al /dev/disk/by-label | grep -E /sr[0-9]+ | wc -l");
+            for my $n (0 .. $srx - 1) {
+                $defaultrepo .= "dvd:/?devices=\${dev:-/dev/sr$n}+";
```

Comment:
Above shell seems only check iso's label if match. It may be removed, since if we set wrong ISO_n as repo, the test would fail as well.
comment in line 91 removed.